templates/kubernetes/compare.html

```diff
@@ -2,470 +2,508 @@
+            </td>
+            <td data-heading="Red Hat Openshift" class="u-align--center">
+              <span data-heading="No">&ndash;</span>
+            </td>
+            <td data-heading="Rancher" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+          </tr>
+          <tr>
+            <td data-heading="Category">Edge support</td>
+            <td data-heading="Canonical Kubernetes">
+              <div class="u-align--center">
+                <span class="p-rating p-rating--5">5 out of 5</span>
+                <p>MicroK8s</p>
+              </div>
+            </td>
+            <td data-heading="Red Hat Openshift">
+              <div class="u-align--center">
+                <span class="p-rating p-rating--4">4 out of 5</span>
+                <p>Openshift</p>
+              </div>
+            </td>
+            <td data-heading="Rancher">
+              <div class="u-align--center">
+                <span class="p-rating p-rating--5">5 out of 5</span>
+                <p>K3s</p>
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <td data-heading="Category">Single-node edition</td>
+            <td data-heading="Canonical Kubernetes" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+            <td data-heading="Red Hat Openshift" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+            <td data-heading="Rancher" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+          </tr>
+          <tr>
+            <td data-heading="Category">Managed Kubernetes offering</td>
+            <td data-heading="Canonical Kubernetes" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+            <td data-heading="Red Hat Openshift" class="u-align--center">
+              <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                   alt="Yes" />
+            </td>
+            <td data-heading="Rancher" class="u-align--center">
+              <span data-heading="No">&ndash;</span>
+            </td>
+          </tr>
+          <tr>
+            <td data-heading="Category">Container runtime and registries</td>
+            <td data-heading="Canonical Kubernetes">
+              <div class="u-align--center">
+                <span class="p-rating p-rating--5">5 out of 5</span>
+                <p>ContainerD, Docker, Kata</p>
+                <p>Private registries, DockerHub, Rocks, public cloud registries</p>
```

Comment:
```suggestion
                <p>Private registries, DockerHub, rocks, public cloud registries</p>
```
Lowercase on the doc